gcn/notices/core/Alert.schema.json

```diff
@@ -26,7 +26,7 @@
         "update",
         "retraction",
         "earlywarning",
-        "premliminary"
+        "preliminary"
```

Comment:
```suggestion
```

@lpsinger, in meeting we decided to remove "earlywarning" & "preliminary" enums. @jracusin @dakota002